homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -1 +1,79 @@
 """Support for Xiaomi Miio."""
+import logging
+
+from miio import DeviceException, gateway
+
+from homeassistant import config_entries, core
+from homeassistant.const import CONF_HOST, CONF_TOKEN
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+GATEWAY_PLATFORMS = ["alarm_control_panel"]
+
+
+async def async_setup(hass: core.HomeAssistant, config: dict):
+    """Set up the Xiaomi Miio component."""
+    conf = config.get(DOMAIN)
```

Comment:
I think this function is called after the Config Flow has finisched and the config includes the user_input of the config flow

Reply:
No. The config passed here is the config yaml.

Reply:
alright, what schould I then do in the async_setup function?
Before I did not have that function at all, but then it would give me an error during setup that that function is missing and the integration would not be setup correctly.

Schould I just have an empty function that imediatly returns True?

Reply:
Yes, just `return True`. :+1: